api/lib/opentelemetry/metrics/meter.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Metrics
+    # No-op implementation of Meter.
+    module Meter
+      extend self
+
+      NOOP_DOUBLE_MEASURE = DoubleMeasure.new
+      NOOP_LONG_MEASURE = LongMeasure.new
+
+      private_constant(:NOOP_DOUBLE_MEASURE, :NOOP_LONG_MEASURE)
+
+      def record(*measurements, distributed_context: nil, exemplar: nil)
+        raise ArgumentError if measurements.empty?
```

Comment:
I think we should drop this check. It's conceivable that a client app could send us an empty list (e.g. if they're dynamically collecting a set of measurements, and happen not to have any collected at a certain point—we avoid them having to wrap this method call in a conditional.) The Java client checks for null but not empty list.

Reply:
Thanks for catching that.